src/main/telemetry/smartport.c

```diff
@@ -78,7 +81,11 @@ enum
 enum
 {
     FSSP_START_STOP = 0x7E,
+    FSSP_BYTE_STUFF = 0x7D,
+
     FSSP_DATA_FRAME = 0x10,
+    FSSP_MSPC_FRAME = 0x30, // MSP client frame
```

Comment:
100%!!! You could use anything you want, as long as you control the apps at both ends. LUA lets you send anything you want as well. The new Frsky S6R receiver uses the same IDs for config packages sent to it.